operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -1,18 +1,16 @@
 # Documentation Style Guide
 
-When writing for Mattermost, always think about the audience that will be reading your writing. Mattermost’s 
-audience can vary, but we typically communicate with a technical audience and want to present Mattermost as 
-being a friend of the practitioner in development, IT, and operations. It must be accurate and clear, be 
-presented with a style and tone that is appropriate for technical content, and easy to navigate.
+When writing for Mattermost, always think about the audience that will be reading your writing. Mattermost’s audience can vary, but we typically communicate with a technical audience and want to present Mattermost as being a friend of the practitioner in development, IT, and operations. It must be accurate and clear, be presented with a style and tone that is appropriate for technical or educational content, and easy to navigate.
 
 ## Document structure
 
-Structure and organization are an important part of a document's ease of use and its understandability. 
-Information should be organized and presented in a logical order, with similar subjects grouped together in the 
-same section. In most cases, a document has a title, an introductory paragraph, and one or more sections.
+Structure and organization are an important part of a document's ease of use and its understandability. Information should be organized and presented in a logical order, with similar subjects grouped together in the same section. In most cases, a document has a title, an introductory paragraph, and one or more sections.
 
-Try to keep only one topic in a page. Shorter topics are easier to reuse in other documents, are 
-easier to write and edit, and are easier to translate.
+Try to keep only one topic in a page. Shorter topics are easier to reuse in other documents, are easier to write and edit, and are easier to translate.
+
+### Educational guides
+
+An exception are educational guides. In their abstract they set clear expectations for a well defined audience, can have more then one topic in a page, and do not require an introductory paragraph. First person writing can be used, as they need to keep the reader interested in the subject.
```

Comment:
Let's make the exceptions clear, and add a concise note on why those exceptions are made for educational guides.

E.g. later in this page we have a note about the abstract

```
The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want.
```

I'm wondering how much educational guides deviate from this?

Reply:
The Idea was to write a single paragraph telling before hand any changes to the below sections for someone to write educational content, so they only have to read a single paragraph if they are already  familiar with the style/voice/tone guidelines.

Of course the deviations can be mentioned per paragraph, but someone already familiar with the document would need to reread everything in order to get all differences.